dandischema/models.py

```diff
@@ -979,12 +1076,14 @@ class BareAsset(CommonModel):
 
     wasDerivedFrom: Optional[List[BioSample]] = Field(None, nskey="prov")
     wasAttributedTo: Optional[List[Participant]] = Field(
-        None, description="Participant(s) to which this file belongs to", nskey="prov"
+        None,
+        description="Participant(s) or subject(s) associated with this file.",
```

Comment:
```suggestion
        description="Participant(s) or subject(s) associated with this asset.",
```
who knows where the life brings us, and there is `file`, `asset` and `asset file` used interchangeably through descriptions.  Should we unify it all?  could as well be just
```suggestion
        description="Associated participant(s) or subject(s).",
```
thus avoiding mentioning one of the three (and change the rest to follow)

Reply:
BioSample has it
```
 description="Participant(s) or Subject(s) associated with this sample.",
```
so IMHO could be unified between the two to just that latter form in the comment above